plugins/labhub.py

```diff
@@ -387,3 +388,65 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
```

Comment:
I dont like `corobo pr list` , because this is about mergable PRs only, so the command needs to reflect that.

Reply:
Is `corobo mergable prs` ok?